templates/docs/patterns/icons.md

```diff
@@ -149,6 +149,48 @@ Our alert icons are used to indicate the status of a message in a notification.
   </div>
 </section>
 
+### Application
+
+Outside of the standard set, additional icons are available for application-based projects.
```

Comment:
Because these are our first icons not included by default I would refer to that as well here, explaining that they need to be included separately and either explain how here (shortly) or link to the import section below.